main/zoe/guide/contracts/callSpread.md

```diff
@@ -0,0 +1,77 @@
+financial building block to create futures, puts, calls, and event binaries that would form the
+basis for a prediction market, insurance, and much more. A Call Spread is a combination of a call
+option bought at one strike price and a second call option sold at a higher price. That means there
+are two parties to the call spread, each holding an ERTP asset that pays out complementary amounts
+depending on the value of some good at a known future time.
+
+The Zoe invitations representing the options are produced in pairs that represent the matching
+assets.  The purchaser deposits the entire amount that will be approtioned between the holders of
+the two positions. The individual options are ERTP invitations that are suitable for resale.
+
+This option is settled financially. There is no requirement that the original purchaser have
+ownership of the underlying asset at the start, and the beneficiaries shouldn't expect to take
+delivery at closing.
+
+It is common for the strike price and collateral to use the same currency, though this contract
+decouples them. This would be like a spread based on the price of APPL stock (`Underlying`),
+where the stock price is measured in USD (`Strike`), but the contract pays out in JPY
+(`Collateral`).
+
+The issuerKeywordRecord specifies the issuers for four keywords: Underlying, Strike, Collateral,
+and Quote. The original deposit and the payout are in Collateral. Strike amounts are used for the
+price oracle's quote as to the value of the Underlying, as well as the strike prices in the
+terms. The priceAuthority's quotes are ERTP assets issued by a QuoteAuthority, which must be
+specified using the keyword Quote.
+
+The terms include { timer, underlyingAmount, expiration, priceAuthority, strikePrice1,
+strikePrice2, settlementAmount }. `timer` must be recognized by `priceAuthority`. `expiration` is a
+time recognized by the `timer`. `underlyingAmount` is passed to `priceAuthority`, so it could be an
+NFT or a fungible amount. `strikePrice2` must be greater than `strikePrice1`. `settlementAmount`
+uses Collateral.
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#startInstance
+
+The terms specify all the details of the options that will be created, but the options are not
+handed out until the creator funds the collateral that will make them valuable.  The
+creatorInvitation has customProperties that include the amounts of the two options as longAmount
+and shortAmount.
+
+<<< @/snippets/zoe/contracts/test-callSpread.js#invitationDetails
+
+The funder can use these option amounts to create an offer that assures them they will get the two
+options in exchange for the funds. The proposal describes the options they want and the collateral
+they give. When the offer is made they get a payout that contains the two option positions, which
+are themselves invitations which can be exercised for free, and provide the option payouts with the
+keyword Collateral.
```

Comment:
```suggestion
The funder can use these option amounts to create an offer assuring they will get the two options in exchange for the funds. The proposal describes the wanted options and the collateral they give. When the offer is made, they get a payout containing the two option positions. The positions are invitations which can be exercised for free, and provide the option payouts with the keyword-specified Collateral.
```

Reply:
In "When the offer is made, they", "they" is unclear in what it refers to. The use of "a payout" following implies "they" is a singular thing, but since the preceding uses of "they" referred to plurals, you should drop the pronoun and say just who/what is getting the payout (I think it's the funder, since that's the only singular actor given).

Reply:
I reworded here and changed that to "when the offer is made, a payout is returned".